_specs/ecip-1000.md

```diff
@@ -353,6 +355,19 @@ In addition, it is recommended that literal code included in the ECIP be dual-li
 * Some ECIPs, especially consensus layer, may include literal code in the ECIP itself which may not be available under the exact license terms of the ECIP.
 * Despite this, not all software licenses would be acceptable for content included in ECIPs.
 
+# ECIP Process Code of Conduct
+
+## ECIP Participants Will Adhere to the Following
+
+1. Anyone can participate in the ECIP process as a competent developer, miner, validator, node operator, user, or any other prescribed participant or stakeholder.
+2. Within the ECIP documents and discussion pages, participants will conduct themselves professionally, focusing on ECIP issues, making sure their arguments are well founded, minimizing trolling, insulting or derogatory comments, and personal or political attacks.
+3. Participant privacy will be respected. Publishing other participant’s private information, such as a physical or email addresses, or real names, without their explicit permission is not allowed within the ECIP process nor in other public forums.
+4. Ethereum Classic organization owners on Github or other platform in use, ECIP editors, and ECIP coordinators have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to ECIP-1000 and Code of Conduct, and will communicate reasons for moderation decisions when appropriate.
```

Comment:
I was gonna bring up things about how to resolve conflicts but doing so on an organization-owner level is pretty clever because it enforces localism within ETC. Makes everyone self-moderate.